fixtures/analytics/query_all_analytics_fixtures.py

```diff
@@ -25,10 +33,15 @@
             totalBookings
             period
           }
+          deviceAnalytics{
+            deviceName
+            deviceId
+            downTime
+          }
         }
   }
-'''
-all_analytics_query_invalid_locationid = '''
+"""
```

Comment:
Good work on the PR @raymond42, although I do not find any reason to change the single quotes to double and also the formatting of some queries.

Reply:
@shaluchandwani I added some lines of code in this file and when I pushed flake complained and I used black module to format in an advised way, and that formated everything in the file, which are the changes you are seeing. Thanks